ci/Makefile

```diff
@@ -21,6 +21,13 @@ endif
 ifdef KUBERNETES_VERSION
 K8S_VERSION_OPT=--kubernetes-version ${KUBERNETES_VERSION}
 endif
+ifdef BRANCH_REPO
+BRANCH_REGISTRY=$(subst https://download.suse.de/ibs/,registry.suse.de/,${BRANCH_REPO})
```

Comment:
I'm not sure we need this logic for creating the branch repository url. As we know in advance the format of the URLs, we could extract the branch name from the branch repo's url and use it in the branch registry url.

Reply:
Changed. Do you prefer this version?

Reply:
Yes, it is cleaner.